service/create/encrypt.go

```diff
@@ -0,0 +1,149 @@
+package create
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awssession "github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/giantswarm/k8scloudconfig"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+// PEM encoded TLS assets.
+type rawTLSAssets struct {
+	APIServerCACrt    []byte
+	APIServerKey      []byte
+	APIServerCrt      []byte
+	CalicoClientCACrt []byte
+	CalicoClientKey   []byte
+	CalicoClientCrt   []byte
+	EtcdServerCACrt   []byte
+	EtcdServerKey     []byte
+	EtcdServerCrt     []byte
+}
+
+// Encrypted PEM encoded TLS assets
+type encryptedTLSAssets struct {
+	APIServerCACrt    []byte
+	APIServerKey      []byte
+	APIServerCrt      []byte
+	CalicoClientCACrt []byte
+	CalicoClientKey   []byte
+	CalicoClientCrt   []byte
+	EtcdServerCACrt   []byte
+	EtcdServerKey     []byte
+	EtcdServerCrt     []byte
+}
+
+func readRawTLSAssets(tlsAssetsDir string) (*rawTLSAssets, error) {
+	r := new(rawTLSAssets)
+	files := []struct {
+		name          string
+		ca, key, cert *[]byte
+	}{
+		{"apiserver", &r.APIServerCACrt, &r.APIServerKey, &r.APIServerCrt},
+		{"calico/client", &r.CalicoClientCACrt, &r.CalicoClientKey, &r.CalicoClientCrt},
+		{"etcd/server", &r.EtcdServerCACrt, &r.EtcdServerKey, &r.EtcdServerCrt},
+	}
+	for _, file := range files {
+		caPath := filepath.Join(tlsAssetsDir, file.name+"-ca.pem")
+		keyPath := filepath.Join(tlsAssetsDir, file.name+"-key.pem")
+		certPath := filepath.Join(tlsAssetsDir, file.name+".pem")
```

Comment:
This is not that important and should not be fixed in this PR. Anyway, because I see it now. I remember we agreed on naming the certificate files according to a certain scheme. 

```
foo-ca.pem
foo-crt.pem
foo-key.pem
```

The according variables should follow the scheme somehow. 

```
caFile
crtFile
keyFile
```

Lets create an issue so we tackle this at some point and have it tracked in the boards. It is not that important priority wise. It is more important to get the AWS operator functional. Just keep in mind schemes we agree on while you implement functionality. <3